modules/rosa-sts-support-considerations.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started_sts/rosa_creating_a_cluster_with_sts/rosa-sts-creating-a-cluster-with-customizations.adoc
+
+[id="rosa-sts-support-considerations_{context}"]
+= Support considerations for ROSA clusters with STS
+
+The supported way of creating a {product-title} cluster that uses the AWS Security Token Service (STS) is by using the steps described in this product documentation. Do not use other configurations, as they are unsupported.
+
+You can use `manual` mode with the {product-title} CLI (`rosa`) to generate the AWS Identity and Access Management (IAM) policy files and `aws` commands that are required to install the STS resources. The files and `aws` commands are generated for review purposes only and must not be modified in any way. Red Hat cannot provide support for ROSA clusters that have been deployed by using modified versions of the policy files or `aws` commands.
```

Comment:
Instead of "You can use" lets just state "The use of `manual` mode [...] will generate [...]"